src/yaml-cfn/nunjucks-parse-includes.ts

```diff
@@ -0,0 +1,25 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+import { nunjucksParse } from '.';
+
+const include = /!Include\s+('|")?([^'"\s]*)('|")?/g;
+export const nunjucksParseContentWithIncludes = (contents: string, directory: string, filename: string, data: any): any => {
```

Comment:
see previous comments on separating text-templating from the parser

Reply:
done